bin/helm-docs-diff

```diff
@@ -10,6 +10,6 @@ if [ -z "$dir_dirty" ]; then
   exit 0
 else
   echo "Helm-docs generated readmes diverge from current chart readmes:"
```

Comment:
Single quotes are preferred here since nothing is to be expanded.

(also goes for line 9)